Bender.yml

```diff
@@ -0,0 +1,31 @@
+package: 
+  name: fpu_interco
+  authors: 
+    - "Igor Loi <igor.loi@greenwaves-technologies.com>"
+    - "Stefan Mach <smach@iis.ee.ethz.ch>"
+    - "Florent Rotenberg <florent.rotenberg@greenwaves-technologies.com>"
+    - "Andreas Kurth <akurth@iis.ee.ethz.ch>"
```

Comment:
```suggestion
```
That's undeserved glory ;-) I did not contribute hardware code to this repository.

Reply:
I went by contributors to the github repository, but will update